CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - Fixed: bug causing rules in extended configs to be merged with, rather than replaced by, the extending config.
 - Fixed: bug where fractional keyframes yielded false positives in `selector-class-pattern`
 - Fixed: `selector-no-type` with secondary option `ignore: ["descendant"]` will now resolve nested selectors.
+- Fixed: `selector-max-specificity` now ignores selectors containing pseudo-classes with functional notation like `:unknown()`, and warns if the underlying `specificity` module cannot parse the selector. 
```

Comment:
We did have a try catch block for all the pseudo elements right? 
I changed the message from that to this.